devops/docker/idea-image/ideaIC-2022.2.2/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM cimg/openjdk:11.0
```

Comment:
>devops/docker/idea-image/ideaIC-2022.2.2/Dockerfile

Maybe devops/docker/idea-image/Dockerfile
We will reuse this file again and again during upgrade of idea version.
Docker hub will store all versions of containers.
During deploy to hub I will use tag like ideaIC-2022.2.2

Reply:
ok, I am good with this. Please push all tags when tagging so that we have 'latest' and idea version tag.